src/connectors/playirish.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+Connector.playerSelector = '.qt-musicplayer';
+
+Connector.artistTrackSelector = '#streamCurrentlyPlaying';
+
+Connector.getArtistTrack = () => {
+	let text = $(Connector.artistTrackSelector).text().replace('NOW PLAYING:', '');
+	return Util.splitArtistTrack(text);
+};
+
+Connector.isPlaying = () => $('.sm2_playing').length > 0;
+Connector.playButtonSelector = '.sm2_playing';
```

Comment:
There's no sense to use `Connector.playButtonSelector` if `Connector.isPlaying` is overriden. You should use only one of them; but selectors is preferrable in most cases.